Northstar.Client/mod/scripts/vscripts/ui/atlas_auth.nut

```diff
@@ -37,7 +37,7 @@ void function AtlasAuthDialog_Threaded()
 	if ( res.errorCode != "" )
 		dialogData.message += format( "\n\n%s", Localize( "#AUTHENTICATION_FAILED_ERROR_CODE", res.errorCode ) )
 
-	string link = "https://r2northstar.gitbook.io/r2northstar-wiki/installing-northstar/troubleshooting"
+	string link = "https://docs.northstar.tf/Wiki/installing-northstar/troubleshooting/"
```

Comment:
Is there a reason to add trailing slashes? doesn't matter though

Reply:
MkDocs redirects to the page with the trailing slash if the called URL doesn't have one ^^

Reply:
Ah so it just removes a needless redirect? Fair enough